lib/models/lookup.js

```diff
@@ -58,7 +57,7 @@ function LookupModelFactory (
             },
             {
                 keys: { ipAddress: 1},
-                options: { unique: true }
+                options: { unique: false }
```

Comment:
Thanks @cgx027 : I need to check @sunnyqianzhang 's system to see whether the duplicated ipAddress is sure added into RackHD, may be this is another problem. I will check before merging.